__tests__/test-clone.js

```diff
@@ -112,4 +112,22 @@ describe('clone', () => {
     )
     expect(fs.existsSync(`${dir}/a.txt`)).toBe(true, `'a.txt' exists`)
   })
+
+  it.only('errors if remote repository is empty', async () => {
```

Comment:
😆 That's twice now. You just really want your tests to be the only ones, don't you!

Reply:
:man_facepalming: